dev-ml/llvm-ocaml/llvm-ocaml-9999.ebuild

```diff
@@ -28,10 +28,11 @@ SLOT="0/${PV}"
 KEYWORDS=""
 IUSE="test ${ALL_LLVM_TARGETS[*]}"
 
+LLVM_SLOT="5"
```

Comment:
Not in `RDEPEND` scope. Maybe I could just use `:=` since it is already bound to a specific version. The idea behind specific slot was that 4.0.0.rc1 was already in :0, and I suspect that people will want slotting backported to 3.9.1. I guess we don't have to babysit people using the live ebuild that much, and just get the slot from llvm-config afterwards.

That said, I missed a blocker against `llvm-ocaml:0`.